course_content/notebooks/cartopy_intro.ipynb

```diff
@@ -15,7 +15,24 @@
    "cell_type": "markdown",
    "metadata": {},
    "source": [
-    "## Cartopy in a nutshell\n",
+    "# Cartopy in a nutshell\n",
+    "\n",
+    "## Course aims and objectives\n",
+    "\n",
+    "The **aim** of the cartopy course is to introduce the cartopy library and raise awareness of some of its features.\n",
```

Comment:
'raise awareness' makes it sound a bit like a disease.  Maybe use 'highlight' or something instead?

Reply:
😷